controller/space_codebases.go

```diff
@@ -58,6 +60,9 @@ func (c *SpaceCodebasesController) Create(ctx *app.CreateSpaceCodebasesContext)
 		}
 		err = appl.Codebases().Create(ctx, &newCodeBase)
 		if err != nil {
+			if gormsupport.IsUniqueViolation(err, "codebases_spaceid_url") {
```

Comment:
for some reason this error is not being parsed to be of type `gormsupport.IsUniqueViolation`, investigating that as of now